libs/designsystem/calendar/src/calendar.component.ts

```diff
@@ -354,19 +351,27 @@ export class CalendarComponent implements OnInit, AfterViewInit, OnChanges {
       const day = this.getCalendarDay(cellDate, today, monthStart);
 
       const isSelectable = this.isSelectable(day, cellDate);
+      const isFocusable = this.isFocusable(cellDate, today);
       const isSelected = isSameDay(this.selectedDate, cellDate);
+      const isFocussed = isSameDay(this.focussedDate, cellDate);
       const cell: CalendarCell = {
         date: cellDate.getDate(),
         monthIndex: cellDate.getMonth(),
         year: cellDate.getFullYear(),
         isCurrentMonth: day.isCurrentMonth,
         isSelectable,
+        isFocussed,
         isSelected,
-        cssClasses: this.getCssClasses(day, isSelectable, isSelected),
+        isFocusable,
+        ariaLabel: this.formatWithLocale(cellDate, 'd MMMM'),
```

Comment:
Should the format string (order of day and month) be the same regardless of locale? And should it contain punctuation? 🤔 
I.e.:
* en-GB: "4 November
* en-US: ""November 4"
* da: "4. november"

https://englishlessonsbrighton.co.uk/how-to-write-dates-british-american-english/

Reply:
Good point, I added functionality for this in ea547b6